matcher/service/openreview_interface.py

```diff
@@ -165,6 +171,10 @@ def paper_notes(self):
     def papers(self):
         return [note.id for note in self.paper_notes]
 
+    @property
+    def map_papers_to_indexes(self):
+        return {p:i for i,p in enumerate(self.papers)}
```

Comment:
I was talking about having this as a property of the class, not a setter method. This dictionary is going to be created every time we call `map_papers_to_indexes`

Reply:
I see. I have fixed it now.